src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/CoroutineCancellationTraceReportingTest.kt

```diff
@@ -17,6 +17,7 @@ import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.ModelChecki
 import org.jetbrains.kotlinx.lincheck_test.util.*
 import org.junit.*
 
+@IgnoreInTraceDebuggerMode // TODO investigate (Evgeniy Moiseenko)
```

Comment:
Can we not ignore but have a different output file when `inTraceDebuggerMode == true`?

Reply:
Several coroutine-related tests have different trace when run without local object detector. 
However, as we discussed with @zhelenskiy it could actually be due to a bug in local objects detector. 
Hence the comment:
```
TODO investigate (Evgeniy Moiseenko)
```

Reply:
@eupp, would it be better to continue running the test, thus checking that the framework does not fail on this code, and use another output file for the verification (as we do for different JDKs)?

Reply:
I don't know. At least, let's create an issue then with the list of tests, so we don't forget to investigate it later.